### PR TITLE
Update hooks when changes happens on RSVP and tickets

### PR DESCRIPTION
**Pull request by mitogh**
Ticket: https://central.tri.be/issues/123732

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:41](https://github.com/moderntribe/event-tickets/blob/70e20d36d24f60af27bb8ce04216f436b43922cf/src/Tribe/Promoter_Observer.php#L41)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Promoter_Observer.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		add_action( 'save_post_' . $event_type, array( $this, 'notify' ), 10, 1 );
+	}
+
+	public function notify_event_id( $ticket_id, $event_id ) {
+		$this->notify( $event_id );
+	}
+
+	public function notify( $post_id ) {
+		/** @var Tribe__Promoter__Connector $connector */
+		$connector = tribe( 'promoter.connector' );
```
Comment (sc0ttkclark) [inline src/Tribe/Promoter_Observer.php]:
```suggestion
		$connector = tribe( 'promoter.connector' );
		
		if ( ! $connector ) {
			return;
		}

```

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:41](https://github.com/moderntribe/event-tickets/blob/01a6d8f35d93a5d9646feaf5527a3facc3af5ac9/src/Tribe/Promoter_Observer.php#L41)
:x: | Whitespace found at end of line | [Promoter_Observer.php:48](https://github.com/moderntribe/event-tickets/blob/01a6d8f35d93a5d9646feaf5527a3facc3af5ac9/src/Tribe/Promoter_Observer.php#L48)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:64](https://github.com/moderntribe/event-tickets/blob/dd1d335da03513a26b309f6d1b605c9e1b477924/src/Tribe/Promoter_Observer.php#L64)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:64](https://github.com/moderntribe/event-tickets/blob/c9dcfc6e77d687adbbe113c35faf181f832a32bd/src/Tribe/Promoter_Observer.php#L64)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:64](https://github.com/moderntribe/event-tickets/blob/0656401b522b50feca6be383122e823115013fab/src/Tribe/Promoter_Observer.php#L64)
:warning: | Comment refers to a TODO task &quot;Report this to the logger&quot; | [Promoter_Observer.php:79](https://github.com/moderntribe/event-tickets/blob/0656401b522b50feca6be383122e823115013fab/src/Tribe/Promoter_Observer.php#L79)

> via jenkins/codesniffer-phpcs

File: readme.txt [hunk 2]
```diff
@@ -124,7 +124,7 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [4.10.1.1] 2019-03-06 =
 
-
+* Feature - Add hooks to notify promoter when an event with tickets has changes
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
* Feature - Add hooks to notify Promoter when an event with tickets has changes
```

File: src/Tribe/Main.php [hunk 3]
```diff
@@ -295,6 +296,8 @@ public function bind_implementations() {
 
 		// Privacy
 		tribe_singleton( 'tickets.privacy', 'Tribe__Tickets__Privacy', array( 'hook' ) );
+		// Promoter Observer
```
Comment (sc0ttkclark) [inline src/Tribe/Main.php]: This and the tribe() call probably need to be in Service_Provider.php

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:64](https://github.com/moderntribe/event-tickets/blob/5d71e076b63ba91418503fac83ae6f2b1cdfb440/src/Tribe/Promoter_Observer.php#L64)
:warning: | Comment refers to a TODO task &quot;Report this to the logger&quot; | [Promoter_Observer.php:79](https://github.com/moderntribe/event-tickets/blob/5d71e076b63ba91418503fac83ae6f2b1cdfb440/src/Tribe/Promoter_Observer.php#L79)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $ticket_id is never used | [Promoter_Observer.php:64](https://github.com/moderntribe/event-tickets/blob/e709b5d264f1549ae28b221d081d55527c74e000/src/Tribe/Promoter_Observer.php#L64)
:warning: | Comment refers to a TODO task &quot;Report this to the logger&quot; | [Promoter_Observer.php:79](https://github.com/moderntribe/event-tickets/blob/e709b5d264f1549ae28b221d081d55527c74e000/src/Tribe/Promoter_Observer.php#L79)

> via jenkins/codesniffer-phpcs

**Pull request merged**